src/main/java/edu/upf/taln/dri/lib/model/ext/HeaderImpl.java

```diff
@@ -261,13 +261,23 @@ public void setInstitutions(List<Institution> institutions) {
 	// Other methods
 	public void addAuthor(Author auth) {
 		if(auth != null) {
-			if(this.authorList == null) {
+			if (this.authorList == null) {
 				this.authorList = new ArrayList<Author>();
 			}
 			this.authorList.add(auth);
 		}
 	}
 
```

Comment:
Each extractor returns the list of all extracted authors, so we added a new method to could add the whole return statement at once using the above method as reference.